### PR TITLE
Setup update

### PR DESCRIPTION
**Pull request by jGaboardi**
Addressing #11

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,17 +23,18 @@ def setup_package():
...
 
-    setup(name='spopt', #name of package
+    setup(name=package,
           version=__version__,
           description='', #short <80chr description
-          url='', #github repo
+          url='https://github.com/pysal/'+package, #github repo
           maintainer='',
           maintainer_email='',
           test_suite = 'nose.collector',
```
Comment (knaaptime) [inline setup.py]: i dont know that it matters but we're using pytest rather than nose

Comment (knaaptime) [reply setup.py]: or, we'd intended to anyway. Not sure an official decision has been made since we dont have any tests yet

Comment (jGaboardi) [reply setup.py]: it probably does matter... i'll make the switch.

Comment (jGaboardi) [reply setup.py]: @knaaptime would this simply be `test_suite = 'pytest'`?

**Comment by jGaboardi** [comment]
OK, well, maybe we'll leave `test_suite = 'nose.collector'` as it is for now and change later once we decide on testing.

**Pull request merged**